tools/test/h5diff/h5diffgentest.c

```diff
@@ -8929,3 +8940,135 @@ write_dset(hid_t loc_id, int rank, hsize_t *dims, const char *name, hid_t tid, v
 
     return FAIL;
 } /* end write_dset() */
+
+/* TBD: Duplicated from h5dumpgentest */
+static void
```

Comment:
We probably need to think of a better solution for files which are generated by one generator program and used by multiple tools. CMake can't necessarily copy those files as needed, but duplicating the code to generate the files means you have to know and remember to update multiple places when you change one of the testing files.

Reply:
I believe the VOL tools tests originally used repack to read the files in with the native VOL and repack them to the VOL connector set in `HDF5_VOL_CONNECTOR`.